webpack.config.js

```diff
@@ -176,7 +176,7 @@ if (config.mode === 'production') {
 const symbolication_tool_config = {
   name: 'symbolication-tool',
   target: 'node',
-  mode: 'development',
+  mode: process.env.NODE_ENV,
```

Comment:
I think you can remove this line as webpack picks NODE_ENV automatically (according to their doc)

Reply:
I was mimicking what I saw on line 28. If it doesn't need to be here, I can remove it, just noting it in case line 28 should be removed as well.

Reply:
mmm if it's there I'd say there's a reason polished by years of debugging, so let's keep it :-)